plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6989,7 +6989,7 @@ public void testIdentityPartitionFilterMissing()
     {
         String tableName = "test_partition_" + randomNameSuffix();
 
-        Session session = sessionWithPartitionFilterRequirement();
+        Session session = withPartitionFilterRequired(getQueryRunner().getDefaultSession());
```

Comment:
You can use just `getSession()` instead of `getQueryRunner().getDefaultSession()`

Reply:
Nice, that's much handier